### PR TITLE
Missing endpoints

### PR DESCRIPTION
**Pull request by kiniry**
Here are initial/template endpoint implementations for all missing inbound public event endpoints.

### CODE REVIEW

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+   * Session query parameters: <tt>random-seed</tt>
+   * 
+   * @param the_request The request.
+   * @param the_response The response.
+   */
+  @Override
+  public synchronized String endpoint(final Request the_request, 
+                                      final Response the_response) {
+    ok(the_response, "Contests selected");
+    // try {
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java]: Did you want to commit this commented code?

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/EstablishAuditBoard.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import us.freeandfair.corla.model.UploadedFile;
+import us.freeandfair.corla.model.UploadedFile.FileType;
+import us.freeandfair.corla.model.UploadedFile.HashStatus;
+import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.query.CountyDashboardQueries;
+import us.freeandfair.corla.util.FileHelper;
+import us.freeandfair.corla.util.SparkHelper;
+
+/**
+ * The "ballot manifest upload" endpoint.
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/EstablishAuditBoard.java]: This is accidentally pasted from elsewhere, right?

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AuditInvestigationReport.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+import spark.Response;
+
+import us.freeandfair.corla.Main;
+import us.freeandfair.corla.model.CastVoteRecord;
+import us.freeandfair.corla.model.CastVoteRecord.RecordType;
+import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.util.SuppressFBWarnings;
+
+/**
+ * The "audit CVR upload" endpoint.
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AuditInvestigationReport.java]: Is this right?

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+import spark.Response;
+
+import us.freeandfair.corla.Main;
+import us.freeandfair.corla.model.ContestToAudit;
+import us.freeandfair.corla.model.DepartmentOfStateDashboard;
+import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.query.DepartmentOfStateDashboardQueries;
+
+/**
+ * The endpoint for selecting the contests to audit.
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java]: I think this needs updating.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  protected ASMEvent endpointEvent() {
+    return INDICATE_FULL_HAND_COUNT_CONTEST_EVENT;
+  }
+  
+  /**
+   * Attempts to select contests for audit.
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java]: This comment needs updating.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+   */
+  @Override
+  protected ASMEvent endpointEvent() {
+    return INDICATE_FULL_HAND_COUNT_CONTEST_EVENT;
+  }
+  
+  /**
+   * Attempts to select contests for audit. 
+   * 
+   * Session query parameters: <tt>random-seed</tt>
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java]: This should be removed.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java (hunk 1 above)
Comment (kiniry) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java]: Yes, it was copied from another endpoint and does not actually live here, but the code we have to write will be modeled on it somewhat.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/EstablishAuditBoard.java (hunk 2 above)
Comment (kiniry) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/EstablishAuditBoard.java]: Well, accidentally, then I missed updating them, yes.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AuditInvestigationReport.java (hunk 3 above)
Comment (kiniry) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AuditInvestigationReport.java]: No, I'm addressing it too.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java (hunk 4 above)
Comment (dmzimmerman) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java]: I had thought that we discussed the protocol for this, @ranweiler, the other day - and that it was the same endpoint for both selecting contests to audit and selecting contests to hand count. Is there a compelling reason for it to be a different endpoint?

Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java]: @dmzimmerman, just saw this. You are right: that _is_ what we discussed, and I think they can and should be the same.

**Comment by dmzimmerman** [comment]
This was completely subsumed by #242, right?

**Comment by ranweiler** [comment]
@dmzimmerman I think so, according to the commit message of 20466be, anyway.

**Comment by dmzimmerman** [comment]
I think so too, both from that and from looking at the comments. So I'm going to close it.

**Pull request closed without merging**